src/main/java/org/candlepin/service/impl/HypervisorUpdateAction.java

```diff
@@ -154,7 +154,7 @@ public Consumer reconcileHost(Owner owner, ConsumerDTO incomingHost, HypervisorU
             null;
 
         String hypervisorId = incomingHost.getHypervisorId().getHypervisorId();
-        Consumer resultHost = consumerCurator.getExistingConsumerByHypervisorIdOrUuid(owner.getId(),
+        Consumer resultHost = getExistingConsumerByHypervisorIdOrUuid(owner.getId(),
```

Comment:
We can take this cleanup a few steps further:
- We know the config doesn't change from moment-to-moment, so there's no reason to be constantly pulling the `USE_SYSTEM_UUID_FOR_MATCHING` config on every iteration. We can move that to an instance variable and just reference it in the new `getExistingConsumerBy...` method.
- Moving the config check means we don't need to bother with in-line ternary op and can move the check down to 466 between the consumer and systemUuid checks.

Reply:
done